src/lib/core/coio_task.c

```diff
@@ -150,11 +151,13 @@ coio_enable(void)
 	ev_async_init(&coio_manager.coio_async, coio_async_cb);
 
 	ev_async_start(loop(), &coio_manager.coio_async);
+	coio_is_running = true;
```

Comment:
`coio_enable` may be called from any thread, not just tx (note `coio_manager` is thread-local) so setting the global flag here is incorrect.

Reply:
Got rid of this flag in new version.